demos/mosquitto/snapcraft.yaml

```diff
@@ -19,13 +19,13 @@ apps:
 
 parts:
   mosquitto:
-    plugin: copy
+    source: conf
+    plugin: dump
```

Comment:
Ah, right, well then, ignore the "now" word :stuck_out_tongue: . Some of these have source before, some after. I kind of prefer the plugin first since it's what uses the source, but I don't really have strong feelings on anything other than consistency.